network/edge-services/reference-content/understanding-pricing.mdx

```diff
@@ -0,0 +1,169 @@
+
+    <Lightbox src="scaleway-pipelines-limit.webp" alt="In the Edge Services section of the Scaleway console, bar indicators show how many pipelines and how much cache data has been consumed this month" />
+
+- View the details of your current plan, your last monthly invoice, and the costs accumulated so far for Edge Services this month, in the **Plans** tab of your [Edge Services dashboard](https://console.scaleway.com/edge-services)
+
+    <Lightbox src="scaleway-current-conso.webp" alt="In the Edge Services section of the Scaleway console, information displays about your current plan, last monthly invoice, and charges accumulated this month for Edge Services" />
+
+- Use [Scaleway Cockpit](/network/edge-services/how-to/monitor-cockpit/) to monitor the data egressing from your Edge Services caches.
+
+## Included usage vs additional charges
+
+Additional charges apply when you either:
+
+- Have more pipelines in existence (at any given time) than the limit of your monthly plan
+- Egress more data from all your pipelines' caches combined, than the limit of your monthly plan
+
+Read on to understand more about how additional charges are applied.
+
+### Additional pipeline charges
+
+Additional pipeline charges are based on a monthly fee per additional pipeline. This monthly fee is applied pro-rata based on how long you had additional pipelines in existence during the calendar month.
+
+For example:
+
+<Message type="important">
+The example prices and limits used below are subject to change. You should always refer to the [pricing page](https://www.scaleway.com/en/pricing/network/#edge-services) for the most up to date information.
+</Message>
+
+- For the entire month of November, you are subscribed to the **Starter** plan, which has a limit of 1 pipeline.
+- You start the month with 1 pipeline (A) in existence, which runs until the end of the month.
+- On November 5 you create a second pipeline (B), which you later delete on November 10.
+- On November 15 you create another pipeline (C), which runs until the end of the month. 
+
+There are 30 days in the month of November. 
+- Pipeline A falls within the limit of your Starter plan, so is not billed.
+- Pipeline B was an additional pipeline which existed for 5 days
+- Pipeline C is an additional pipeline which existed for 15 days
+
+Your November Edge Services billing, in terms of additional pipeline charges, is therefore calculated as follows:
+
+`({Monthly flat fee for an additional pipeline} / 30 days) * 5 days` + `({Monthly flat fee for an additional pipeline} / 30 days) * 15 days` <br/>
+e.g. `(4 / 30) * 5 = €0.67` +  `(4 / 30) * 15 = €2` = `€2.67`
+
+### Additional cache charges
+
+Cache consumption is based on the volume of data egressed from all your Edge Services pipelines' caches combined, in a given month. The transfer of data from the origin bucket or Load Balancer into the cache does not count towards the limit of your plan, and does not incur any additional charges.
```

Comment:
Actually, we only egress data that is retrieved for the local cache.i i.E. we don't bill the data if fetched from the origin. Maybe it's worth mentioning it?

Reply:
I've updated to "The transfer of data from the origin bucket or Load Balancer into the cache, or directly from the origin to a user, does not count towards the limit of your plan, and does not incur any additional charges." is that OK?